spec/poller_spec.rb

```diff
@@ -6,7 +6,7 @@
   it 'tests invokes callback until stopped' do
     dbl = double('mock callback function')
     allow(dbl).to receive(:call)
-    expect(dbl).to receive(:call).at_least(1000).times
+    expect(dbl).to receive(:call).at_least(500).times
```

Comment:
For some reason, the call frequency is lower when running on Docker, so I reduced this number.